### PR TITLE
Move models from booking

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/main/java/org/dcsa/core/events/repository/CarrierClauseRepository.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.dcsa.core.events.repository;
+
+import org.dcsa.core.events.model.CarrierClause;
+import org.dcsa.core.repository.ExtendedRepository;
+
+import java.util.UUID;
+
+public interface CarrierClauseRepository extends ExtendedRepository<CarrierClause, UUID> {
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/events/repository/CarrierClauseRepository.java:8]: Add `@Repository` on repository interfaces.

File: src/main/java/org/dcsa/core/events/repository/ShipmentCutOffTimeRepository.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.dcsa.core.events.repository;
+
+import org.dcsa.core.events.model.ShipmentCutOffTime;
+import org.dcsa.core.repository.ExtendedRepository;
+import reactor.core.publisher.Flux;
+
+import java.util.UUID;
+
+public interface ShipmentCutOffTimeRepository extends ExtendedRepository<ShipmentCutOffTime, UUID> {
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/events/repository/ShipmentCutOffTimeRepository.java:9]: Ditto

File: src/main/java/org/dcsa/core/events/repository/ShipmentRepository.java [hunk 3]
```diff
@@ -2,10 +2,16 @@
...
 import org.dcsa.core.events.model.Shipment;
 import org.dcsa.core.repository.ExtendedRepository;
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Pageable;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 import java.util.UUID;
 
 public interface ShipmentRepository extends ExtendedRepository<Shipment, UUID> {
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/events/repository/ShipmentRepository.java:12]: Ditto

File: src/main/java/org/dcsa/core/events/model/CarrierClause.java [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package org.dcsa.core.events.model;
+
+import lombok.Data;
+
+@Data
+public class CarrierClause {
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/events/model/CarrierClause.java:6]: Missing `@Table("carrier_clauses")` on class and  `@Id` and  `@Column` annotations on variables.

**Review by douradofrazer — CHANGES_REQUESTED** [review]

File: src/main/java/org/dcsa/core/events/repository/CarrierClauseRepository.java (hunk 1 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/core/events/repository/CarrierClauseRepository.java:10]: Done

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**